exdb/tests.py

```diff
@@ -1,10 +1,15 @@
-from django.test import TestCase
+from django.test import TestCase, Client
 from django.test.runner import DiscoverRunner
 from django.utils.translation import ugettext as _
 from django.contrib.staticfiles.testing import StaticLiveServerTestCase
 from django.conf import settings
 from selenium import webdriver
 from selenium.webdriver.common.by import By
+from django.utils.timezone import now, timedelta
+from django.contrib.auth.models import User
```

Comment:
You should use the [get_user_model() function](https://docs.djangoproject.com/en/1.9/topics/auth/customizing/#referencing-the-user-model) instead of directly referencing the `User` model.